recipes/configs/qwen2_5/0.5B_full_single_device.yaml

```diff
@@ -0,0 +1,72 @@
+# Config for single device full finetuning in full_finetune_single_device.py
+# using a Qwen2.5 0.5B
+#
+# This config assumes that you've run the following command before launching:
+#   tune download Qwen/Qwen2.5-0.5B-Instruct --ignore-patterns None
+#
+# To launch on a single device, run the following command from root:
+#   tune run full_finetune_single_device --config qwen2_5/0.5B_full_single_device
+#
+# You can add specific overrides through the command line. For example
+# to override the checkpointer directory while launching training:
+#   tune run full_finetune_single_device --config qwen2_5/0.5B_full_single_device checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#
+# This config works only for training on single device.
+
+# Model arguments
+model:
+  _component_: torchtune.models.qwen2_5.qwen2_5_0_5b
```

Comment:
I understand that you're making the filename change 0_5B -> 0.5B for consistency with other configs, but honestly would prefer to just move everything to 0_5B so it matches the builders

Reply:
for llama3.2 we added 3.2 to the path like you did

<img width="1087" alt="image" src="https://github.com/user-attachments/assets/6ae85d5d-e0fd-4d42-85e2-032432e2090b">

Reply:
This is the same thing we have here, no?